test/testdata/argslen.go

```diff
@@ -0,0 +1,7 @@
+//args: -Eargslen
+//config: linters-settings.argslen.maxArguments=1
```

Comment:
@ldez recommendation is to use a config file instead of the annotation/CLI approach.